css/index.css

```diff
@@ -41,7 +47,7 @@ a:hover {
 
 footer {
     color: #999;
-    font-size: .75rem;
+    font-size: .9rem;
```

Comment:
Again, you we need to keep this consistent across all repositories that use this kind of design. File an issue in Concierge (or make a PR).